invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml

```diff
@@ -0,0 +1,178 @@
+    </datafield>
+    <datafield tag="256" ind1=" " ind2=" ">
+      <subfield code="a">Software</subfield>
+      <subfield code="f">FIXME</subfield>
+      <subfield code="t">file</subfield>
+    </datafield>
+    <datafield tag="256" ind1=" " ind2=" ">
+      <subfield code="b">FIXME</subfield>
+      <subfield code="t">bytes</subfield>
+    </datafield>
+    <datafield tag="260" ind1=" " ind2=" ">
+      <subfield code="b">CERN Open Data Portal</subfield>
+      <subfield code="c">2016</subfield>
+    </datafield>
+    <datafield tag="264" ind1=" " ind2="0">
+      <subfield code="c">2016</subfield>
+    </datafield>
+    <datafield tag="516" ind1=" " ind2=" ">
+      <subfield code="a">Use this with the following datasets:</subfield>
+      <subfield code="w">FIXME</subfield>
+      <subfield code="w">FIXME</subfield>
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">Software to produce the intermediate data files, derived from the primary datasets, for a simple analysis on Z decays to two leptons and ZZ decays to four leptons, of CMS 2011 open data. </subfield>
+    </datafield>
+    <datafield tag="538" ind1=" " ind2=" ">
+      <subfield code="a">Software release: CMSSW_5_3_32</subfield>
+      <subfield code="i">Use this code with the CMS Open Data VM environment for 2011 open data</subfield>
+      <subfield code="w">FIXME:-252-</subfield>
+    </datafield>
+    <datafield tag="540" ind1=" " ind2=" ">
+      <subfield code="a">GNU General Public License (GPL) version 3</subfield>
+    </datafield>
+    <datafield tag="556" ind1=" " ind2=" ">
+      <subfield code="a">Only muon and electron collections are kept. Processing the full primary datasets took ? hours.</subfield>
+    </datafield>
+    <datafield tag="567" ind1=" " ind2=" ">
+      <subfield code="a">The software produces the physics object in PAT candidate format. Only muon and electron PAT candidate collections are stored. No further selection, apart from accepting only the validated runs, is applied.</subfield>
+    </datafield>
+    <datafield tag="581" ind1=" " ind2=" ">
+      <subfield code="a">See the run instructions in</subfield>
+      <subfield code="u">https://github.com/katilp/OutreachExercise2011</subfield>
+      <subfield code="y">GitHub</subfield>
+    </datafield>
+    <datafield tag="583" ind1=" " ind2=" ">
+      <subfield code="a">Only the list of validated runs for primary datasets of 2011 are accepted:</subfield>
+      <subfield code="w">1001</subfield>
+    </datafield>
+    <datafield tag="693" ind1=" " ind2=" ">
+      <subfield code="a">CERN-LHC</subfield>
+      <subfield code="e">CMS</subfield>
+    </datafield>
+    <datafield tag="787" ind1=" " ind2=" ">
+      <subfield code="a">The files created are:</subfield>
+      <subfield code="w">FIXME</subfield>
+      <subfield code="w">FIXME</subfield>
```

Comment:
If this refers to the output files (i.e. the pattuples), they are the one that will be uploaded to eospublic by Aida and Jesus